examples/plugin/src/BUILD

```diff
@@ -0,0 +1,32 @@
+load("//kotlin:kotlin.bzl", "kt_jvm_library", "kt_plugin")
+
+kt_plugin(
+    name = "open_for_testing_plugin",
+    id = "org.jetbrains.kotlin.allopen",
+    options = ["annotation=plugin.OpenForTesting"],
```

Comment:
I'm being pedantic... and defensive about the long term maintenance.

Could you make the options in an attr.string_dict?

e.g.
options = {
  "annotation":"plugin.OpenForTesting"
}

and assemble thiem with an equals in the impl.

It's safer to maintain and validate (I know, it sounds weird, but it can be an issue. One doesn't normally consider sanitizing input in build tooling... but, well, it's necessary) if the arguments are split into key/value pairs.

Reply:
I changed the interface, and added a check for equal signs being in the key part of the dict.